docs/semantics.md

```diff
@@ -163,7 +163,27 @@ Additional file cache [behavior](https://cloud.google.com/storage/docs/gcsfuse-c
 
    - If a Cloud Storage FUSE client modifies a cached file or its metadata, then the file is immediately invalidated and consistency is ensured in the following read by the same client. However, if different clients access the same file or its metadata, and its entries are cached, then the cached version of the file or metadata is read and not the updated version until the file is invalidated by that specific client's TTL setting.     
 
-**Note**: 
+**Kernel List Cache**
+
+The Cloud Storage FUSE kernel-list-cache is used to cache the directory listing (output of `ls`). It significantly improves the workload which involves repeated listing. This is recommended to use it with read-only mounts specifically for Serving and Training workload.
+
+By default, the list cache is disabled. It can be enabled by configuring the **--kernel-list-cache-ttl-secs** cli flag or **file-system:kernel-list-cache-ttl-secs** config flag where:
+*   a value of 0 means disabled. This is the default value.
+*   A positive value represents the ttl (in seconds) to keep the directory list response in the kernel page-cache.
+*   -1 to bypass entry expiration and always return the list response from the cache if available.
+
+**Important Points**
```

Comment:
I think we should upfront recommend that for a multi node scnenario, kernel list cache is recommended to be used for primarily read only workload. 

WDYT?

Reply:
Added in the introduction itself.